test/test_autograd.py

```diff
@@ -7954,10 +7957,11 @@ def __torch_dispatch__(cls, func, types, args=(), kwargs=None):
         s = MySubclass(a)
 
         with fwAD.dual_level():
+            # Only the primal has "alias" called on it
             fwAD.make_dual(s, torch.rand_like(s))
             self.assertEqual(counter[0], 1)
             fwAD.make_dual(torch.rand_like(s), s)
-            self.assertEqual(counter[0], 2)
+            self.assertEqual(counter[0], 1)
```

Comment:
I think I realize what is happening here now, my test last time was completely oblivious to what torch function does.

Before this PR __torch_function__ was wrapping the output of torch.rand_like in the subclass, but now that we disable torch function for this subclass, that no longer happens, and the counter is only triggered once as expected.

Reply:
Yep